### ISSUE TITLE
[Request] Auto Label based on moved location or on folder name

### ISSUE THREAD

**Issue by gggirlgeek** [ratanakvlun/deluge-labelplus#25]
I'd like to request that you add an Auto Label option based on the folder name or file location. (I think the folder name would be easier than the entire path.)

Right now Auto Labeling is not working for incoming magnets. However, the torrents do get successfully moved by Deluge when the downloads are complete. If LabelPlus could detect that a file is being moved and label it, the problem with magnets would be solved because the Auto Label option would be applied AFTER the download completes.

So basically, I need an Auto Label option that can be applied after a download is complete. It seems like the only way to do this would be to detect that a file is being moved and label it then.
